modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/AbstractAttribute.java

```diff
@@ -145,4 +147,24 @@ public String removeAttributeProperty(String propertyName) {
 
         return additionalAttributeProperties.remove(propertyName);
     }
+
+    public Map<String, JSONObject> getAttributeJSONProperties() {
+
+        return additionalAttributeJSONProperties;
+    }
+
+    public JSONObject getAttributeJsonProperty(String propertyName) {
+
+        return additionalAttributeJSONProperties.get(propertyName);
+    }
+
+    public void addAttributeJsonProperty(String propertyName, JSONObject jsonObject) {
+
+        this.additionalAttributeJSONProperties.put(propertyName, jsonObject);
+    }
+
+    public JSONObject removeAttributeJsonProperty(String propertyName) {
```

Comment:
Shall we make these method names consistent? Use `JSON` instead of `Json`?

Reply:
Renamed it with [aa912b8](https://github.com/wso2/charon/pull/420/commits/aa912b8076dc71f9476f288885acee6275a5d19c)